src/components/Patient/PatientInfoCard.tsx

```diff
@@ -40,6 +41,13 @@ export const PatientInfoCard = ({
             <PatientHoverCard patient={patient} facilityId={facilityId} />
           </div>
           {children}
+
+          <PLUGIN_Component
+            __name="PatientHeaderActions"
```

Comment:
plugin name is misleading... we have another component called `PatientHeader` and one might assume it's that.

```suggestion
            __name="PatientInfoCardActions"
```

Reply:
changed to PrintIDCardButton